velox/substrait/SubstraitParser.cpp

```diff
@@ -30,6 +30,16 @@ std::shared_ptr<SubstraitParser::SubstraitType> SubstraitParser::parseType(
       nullability = substraitType.bool_().nullability();
       break;
     }
+    case ::substrait::Type::KindCase::kI8: {
+      typeName = "TINYINT";
+      nullability = substraitType.i32().nullability();
```

Comment:
Should it be substraitType.i8() here (if that exists)? Same for case ::substrait::Type::KindCase::kI16 below.

Reply:
Nice catch. Fixed.